src/lib/accessibility/Behaviors/Popup/PopupBehavior.ts

```diff
@@ -0,0 +1,34 @@
+import { Accessibility } from '../../interfaces'
+import * as keyboardKey from 'keyboard-key'
+
+const isButton = props => {
+  return props && props.trigger && props.trigger.props && props.trigger.props.as === 'button'
+}
+
+const PopupBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    trigger: {
+      role: isButton(props) ? undefined : 'button',
```

Comment:
not sure how we should handle these (role and tabindex) 
If it will be for example anchor, then we whould not set tabindex and neither role.

Reply:
yes, you're right. I am not sure for now how better handle it, but it something that we can discuss.